File: G1_PPAI/G1_PPA1_E1/Entidades/RespuestaPosible.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace G1_PPA1_E1.Entidades
{
    public class RespuestaPosible
    {
        //Atributos
        private string descripcion;
        private int valor;

        //Constructor
        public RespuestaPosible (string descripcion, int valor)
        {
            this.descripcion = descripcion;
            this.valor = valor;
        }
        //Metodos
        public string getDescripcionRta()
        {
            return descripcion;
        }
    }
}

File: PPAI_E3/src/main/java/dsi/g1/entidades/RespuestaPosible.java
package dsi.g1.entidades;

public class RespuestaPosible {
    private String descripcion;
    private int valor;

    public RespuestaPosible(String descripcion, int valor) {
        this.descripcion = descripcion;
        this.valor = valor;
    }
    public String getDescripcion(){
        return descripcion;
    }
}
